File: exercicios_algoritmo_logica_programacao/ex_aulapratica3/ex_aulateorica/ex1.py
num1 = int(input("digite o primeiro numero: "))
num2 = int(input("digite o segundo numero: "))
num3 = int(input("digite o terceiro numero: "))

if (num1 > num2):
    maior = num1
else:
    maior = num2
if (num3 > maior):
    maior = num3
print(f"O maior número entre {num1}, {num2} e {num3} é o {maior}")

File: exerciciosC/lista-saladeaula/ex1.c
#include <stdio.h>
#include <locale.h>

int main(){
    setlocale(LC_ALL, "Portuguese");

    float num1, num2;

    printf("Digite o primeiro número: ");
    scanf("%f", &num1);
    printf("Digite o segundo número: ");
    scanf("%f", &num2);

    float soma = num1 + num2;
    float produto = num1 * num2;

    printf("A soma entre os números é: %.2f\nO produto entre os números é: %.2f", soma, produto);

    return 0;
}